Source/SwiftLintBuiltInRules/Rules/RuleConfigurations/RedundantTypeAnnotationConfiguration.swift

```diff
@@ -8,6 +8,8 @@ struct RedundantTypeAnnotationConfiguration: SeverityBasedRuleConfiguration {
     var severityConfiguration = SeverityConfiguration<Parent>(.warning)
     @ConfigurationElement(key: "ignore_attributes")
     var ignoreAttributes = Set<String>(["IBInspectable"])
+    @ConfigurationElement(key: "ignore_type_interfaces")
```

Comment:
Which other elements apart from properties does this include?

If it's only about properties, we could be more specific with the name by calling it `ignore_properties`.

Reply:
Yes, I think it is limited only to that. I was trying to echo the other rule (`explicit_type_interfaces`) by the name, but your suggestion is probably more accurate. I can change it!